checks/security_policy.go

```diff
@@ -41,6 +45,25 @@ func SecurityPolicy(c checker.Checker) checker.CheckResult {
 			}
 		}
 	}
+    // alternatively check repository owner for default community health file
+    r, _, err := c.Client.Repositories.Get(c.Ctx, c.Owner, ".github")
+    if err != nil {
+        return checker.RetryResult(err)
+    }
+    url := fmt.Sprintf("https://raw.githubusercontent.com/%s/.github/{/ref}/SECURITY.md", c.Owner)
```

Comment:
This can get ratelimited, why not use c.Client.Repositories.DownloadContents api for this ?

Reply:
good call.. helps to know the API. :) pushed an update that uses this instead..